wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands/de.md

```diff
@@ -0,0 +1,66 @@
+# Befehle für das Turnier-Management
+
+Die folgenden Chatbefehle sind für die Fernverwaltung von Mehrspieler-Turnierräumen vorgesehen:
+
+- `!mp make <name>` - Erstellt einen Turnierraum mit dem angegebenen Namen. Maximal 4 solcher Räume können erstellt werden.
+  - Dieser Raum ist insofern besonders, da er nicht geschlossen wird, sobald alle Spieler den Raum verlassen haben und er ist passwortgeschützt vor Spielern, die ihm selbst beitreten wollen.
+  - Wenn der Raum nicht mehr weiter genutzt werden soll, benutzte `!mp close`, um den Raum zu schließen.
+- `!mp makeprivate <name>` - Erstellt einen privaten Turnierraum mit dem angegebenen Namen. Dieser Befehl funktioniert wie `!mp make`, aber der Spielverlauf ist nur für den Raumersteller und die Teilnehmer sichtbar.
+- `!mp name <title>` - Ändert den Raumnamen.
+- `!mp invite <username>` - Lädt einen Spieler in den Raum ein.
+  - Bedenke, dass dies *keine* Blockierungen privater Nachrichten umgeht, die im osu!-Client einstellbar sind. D. h., dass dein Turnierpersonal den Spielern mitteilen muss, "Private Nachrichten nur von Freunden" in den osu!-Einstellungen zu deaktivieren.
+- `!mp lock` - Sperrt den Raum, sodass Spieler nicht mehr ihr Team und ihren Slot ändern können.
+- `!mp unlock` - Kehrt die obigen Ausführungen um.
+- `!mp size <size>` - Stellt die Menge an verfügbaren Slots (1-16) im Raum ein.
+- `!mp set <teammode> [<scoremode>] [<size>]` - Stellt verschiedene Raumeigenschaften ein.
+  - `teammode` - 0: Head To Head, 1: Tag Coop, 2: Team Vs, 3: Tag Team Vs
+  - `scoremode` - 0: Score, 1: Accuracy, 2: Combo, 3: Score V2
+- `!mp move <username> <slot>` - Verschiebt einen Spieler innerhalb des Raums zum angegebenen Slot (Startindex 1).
+- `!mp host <username>` - Transferiert die Gastgeberrechte zu diesem Spieler.
```

Comment:
I looked into existing translations for `host` and found the following:
- osu!(stable) uses `Spielleiter`
- web uses `Host`

Unless we're already using `Gastgeber` somewhere on the wiki I would prefer to use one of these 2. Personally lean towards `Host`.

Reply:
`Gastgeber` is used in an unrelated context in [`People/Community_Contributors`](https://osu.ppy.sh/wiki/de/People/Community_Contributors).

I explicitly used `Rechte` here to better distinguish between the player itself and the rights he receives. Importantly, this comes into play on the *clearhost* example. I think it might get confusing depending on what the translation is. Though, I don't have a problem using `Host` instead. I believe it should be preferred since web already uses it.

`Hostrechte`? 🤔

Reply:
Sounds good to me 👍. I personally always used `Host` in such contexts 🤷